app/src/test/java/org/simple/clinic/summary/PatientSummaryUpdateTest.kt

```diff
@@ -50,6 +50,11 @@ class PatientSummaryUpdateTest {
       facilityConfig = FacilityConfig(diabetesManagementEnabled = false)
   )
 
+  private val facilityWithTeleconsultationEnabled = TestData.facility(
```

Comment:
Ugh, we now have a combination of two boolean flags, which might become harder to reason about.

Maybe we should start thinking about representing this as a set of features on the facility instead of flags.

```kotlin
enum class FacilityFeature {
  DiabetesManagement,
  Teleconsultation
}

data class Facility (
  // Rest of the class properties,

  val features: Set<FacilityFeature>
)
```

Anyway, this is just an exploratory discussion. No need to do anything right now.

Reply:
Yeah, I was trying to avoid that. But it didn't seem possible without making significant changes.